tests/plugins/block_added.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+from pyln.client import Plugin
+
+
+plugin = Plugin()
+
+blocks_catched = []
+
+
+@plugin.subscribe("block_added")
+def notify_block_added(plugin, block, **kwargs):
+    global blocks_catched
+    blocks_catched.append(block["height"])
+
+
+@plugin.method("blockscatched")
+def return_moves(plugin):
+    return blocks_catched
```

Comment:
I first read that as “cached”, but should that perhaps be “caught”?

Reply:
Sorry, I don't speak English. I will submit a PR renaming that.